docs/dev/policy_documents/standards-for-tests.md

```diff
@@ -240,9 +239,9 @@ Sample output:
 ★ Install of core/7zip/16.02/20190513161338 complete with 1 new packages installed.
 1..4
 ok 1 7z exe runs
-ok 2 7z exe output mentions expected version 16.02
+ok 2 7z version matches
```

Comment:
I think the previous version of the test title was badly written; but I liked that it expressed exactly what the test does.
I think the new version (`version matches`) is too unspecific.

Maybe it could be `7z is expected version 16.02` or `7z is expected version`.

Bearing in mind that BATS gives you little to no clues for failing tests, I think it's important for the test titles to be self explanatory.

Reply:
I think you're right here, and I gave it some thought.

What we want to do with the test names is express the two values on either side of the equation. In this case, it's important to have a straight line from the user input when the user runs `hab studio run "./7zip/tests/test.sh ${pkg_ident}"` to the name of the test that is returned. This fulfills the design of the element of least surprise -- I passed `${pkg_ident}` as a parameter to the tests, and the tests then told me that THAT parameter was under test. Makes sense to me!